tests/wicked/before_test.pm

```diff
@@ -46,11 +46,13 @@ sub run {
         zypper_call('--quiet in dhcp-server', timeout => 200);
         if (check_var('WICKED', '2nics')) {
             record_info('Info', 'Set up 2nics DHCP and second ip');
-            assert_script_run(sprintf("ip a a %s dev %s", $self->get_ip(type => 'second_card'), $ctx->iface()));
             $self->get_from_data('wicked/dhcp/dhcpd_2nics.conf', '/etc/dhcpd.conf');
         } else {
             $self->get_from_data('wicked/dhcp/dhcpd.conf', '/etc/dhcpd.conf');
         }
+        if (check_var('WICKED', 'basic') || check_var('WICKED', '2nics')) {
```

Comment:
in line 39 we have logic which assigns 'host' IP to first interface . this will overwrite it . maybe you wanted to use `iface2` here ? or if you really want to set 'second_card'  to `iface` then you disable execution of line 39

Reply:
this is done in REF, and it´s assigning an IP alias to `iface` (eth0). REF doesn't have 2 NICs.

Reply:
1. REF also should have two NICs , no matter if it need it or not
2. but even for single NIC case my comment still make sense because line 39 is executed also in REF so fact of modifying same interface twice is still take place

Reply:
as spoked, let's leave it like this as this is adding an ip alias. I will work on the second NIC on REF in another PR.